sql/core/src/test/scala/org/apache/spark/sql/FileBasedDataSourceSuite.scala

```diff
@@ -377,7 +377,7 @@ class FileBasedDataSourceSuite extends QueryTest with SharedSQLContext with Befo
     // TODO: test file source V2 after write path is fixed.
     Seq(true).foreach { useV1 =>
       val useV1List = if (useV1) {
-        "csv,orc"
+        "csv,orc,parquet"
```

Comment:
nit: Use `true` directly (and cut the remaining lines)

Reply:
We still need to specify the value of `useV1List`